app/assets/javascripts/components/edit_preview.es6.jsx

```diff
@@ -27,43 +18,70 @@ class EditPreview extends Component {
         display: 'flex',
         flexFlow: 'column',
         justifyContent: 'center',
+        width: '356px',
+      },
+      form: {
+        display: 'flex',
+        flexFlow: 'column',
+        justifyContent: 'center',
         alignItems: 'center',
-        flex: '1',
         padding: '12px',
+        marginBottom: '18px',
       },
       image: {
         width: '152px',
         height: '152px',
-        marginBottom: '12px',
+        marginTop: '18px',
         borderRadius: '50%',
       },
     };
   }
+
+  componentDidMount() {
+    this.setState({
+      birthday: this.props.student.birthday,
+      firstName: this.props.student.first_name,
+      lastName: this.props.student.last_name,
+    });
+  }
+
+  updateStudent() {
+    console.log('hello');
+  }
+
+  generateHandler(field) {
+    var state = {};
+    return(event) => {
+      state[field] = event.target.value;
+      this.setState(state);
+    };
+  }
+
   render() {
     var student = this.props.student;
     return (
       <div style={this.styles.container}>
-        <img
-          src='https://scontent.fsnc1-1.fna.fbcdn.net/hphotos-xfp1/t31.0-8/11856297_10200932572512494_2256826043885795533_o.jpg'
-          style={this.styles.image} />
-        <FormField
-          title={'First Name'}
-          placeholder={student.first_name}
-          value={this.state.firstName} />
-        <FormField
-          title={'Last Name'}
-          placeholder={student.last_name}
-          value={this.state.lastName} />
-        <FormField
-          title={'Birthday'}
-          placeholder={student.birthday}
-          value={this.state.birthday} />
-        <FormField
-          title={'Age'}
-          placeholder={student.age}
-          value={this.state.age} />
-        <FormButton content={'Cancel'} />
-        <FormButton content={'Save'} />
+        <CardHeader
```

Comment:
The `CardHeader` component is the new `StudentEdit`, except now it's customizable to render a different looking button on the right hand side.